### ISSUE TITLE
"Pause" accessibility requirement

### ISSUE THREAD

**Issue by arangas** [jssor/slider#69]
Hello,

I'm using the slider in a site that has the accessibility requirement of WCAG 2.0 Level A compliance. It works well navigating by keyboard, but I have found an issue satisfying [Guideline 2.2: Provide users enough time to read and use content](http://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html).

The $PauseOnHover option comes close, but really it needs to pause **on focus** for users that can only navigate by keyboard. Is there another way to do this? If not is it possible to add this as a new switch or flag on this existing one?

Thanks!

**Comment by jssor** [comment]
What do you mean 'on focus'?

**Comment by jssor** [comment]
I guess the 'focusin' and 'focusout' events should be detected outside jssor slider.

When you detect the slider is 'focusin', please call jssor_sldier1.$Pause();
And call 'jssor_slider2'.$Play() when the slider is 'focusout'.

**Comment by arangas** [comment]
Based on your advice, I used the following code which works as specified:

```javascript
var jssor_slider1 = new $JssorSlider$('slider', options);

$Jssor$.$AddEvent(jssor_slider1.$Elmt, "focusin", function () {
    console.log('focusin');
    jssor_slider1.$Pause();
});

$Jssor$.$AddEvent(jssor_slider1.$Elmt, "focusout", function () {
    console.log('focusout');
    jssor_slider1.$Play();
});
```
